.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+
+The **petitions process** mentioned in that document is very important to help us prioritize new features based on the needs of our users. There are some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions) to this process for minor changes and bugfixes, but generally speaking it is a good idea to create an issue or a forum thread, or ask in Slack, before undertaking a change.
+
+Also, please be sure to follow our [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) in all interactions with ClassicPress community members.
+
+## Communication channels
+
+We encourage you to join and ask any questions you have about contributing.
+
+- [Slack](https://www.classicpress.net/join-slack/) - great for real-time chat, posting screenshots and asking questions about anything you're stuck on. Or just socializing.
+- [Petitions website](https://petitions.classicpress.net/) - for proposing new features or major changes for consideration by the community.
+- [GitHub issues](https://github.com/ClassicPress/ClassicPress/issues) - for proposing or discussing bugfixes, or minor improvements. Generally it is a good idea to create a petition for anything that may take a significant amount of time or that may have backwards compatibility implications.
+- [Forums](https://forums.classicpress.net/) - a lot of good info and discussion here too
+
+## Review criteria
+
+When evaluating bug fixes and other code changes in pull requests (PRs), we look for these things, ideally all of them:
+
+  1. The change impacts existing ClassicPress users. Otherwise, there are literally thousands of things we could look at, but we need to prioritize our development time. Right now the best tool we have for this is the [petitions site](https://petitions.classicpress.net/).
+  2. The change is not going to break backward compatibility, and has minimal effects on the existing plugin and theme ecosystem. A good way to evaluate the effects of a change on plugins or themes is to do a search on https://wpdirectory.net. (Major changes are also a possibility but require a planning effort around when and how they will be released, as well as agreement from the community per our [democratic process](https://www.classicpress.net/democracy/).
+  3. The change has automated tests.
+  4. We understand the code change very well or can ask questions of someone who understands it very well.
+
+If your change meets all of these criteria then we will most likely accept it.
+
+We take a lot of care with our platform, and ClassicPress is maintained by volunteers in our free time. This means there may be some delays in merging pull requests.
+
+## What to work on?
+
+If you're not sure where to start contributing, here are some ideas:
+
+  - Review and test [existing PRs](https://github.com/ClassicPress/ClassicPress/pulls), especially looking at how well they fit the criteria described above.
```

Comment:
I don't think this sentence does a good job of stating how important reviewing and testing are. We need to be thorough and careful with the changes we make, and the more eyes we can get on our PRs the better.